ossfuzz/compress_fuzzer.cc

```diff
@@ -0,0 +1,22 @@
+#include <stddef.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include "lz4.h"
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
+{
+  size_t const compressed_dest_size = LZ4_compressBound(size);
+  char *const dest_buffer = (char *)malloc(compressed_dest_size);
+
+  int result = LZ4_compress_default((const char*)data, dest_buffer,
```

Comment:
Thanks for this excellent starting point.

I will likely upgrade it so that it can achieve at least a round-trip (successful compression, followed by successful decompression, and src-dst comparison). But I can take over from this point.

Reply:
So what I envisaged was having separate fuzzers; this simple compress one, one that just tries to decompress the input, and any other ones which would exercise interesting paths.

It's fine to just copy any number of fuzzers into the $OUT directory.